schoolopy/main.py

```diff
@@ -2003,14 +2085,29 @@ def get_message(self, message_id):
 
     # Implement search, resource collections, resource templates
 
+    def _like_item(self, pathItems, unlike=False):
+        """
+        Post a like request
+
+        :param pathItems: The path with values to POST to
+        :param unlike: Whether to unlike or like (Default like)
+        :return: Number of likes on the object.
+        """
+        try:
+            return self._post(pathItems, json.dumps({'like_action': True and (not unlike)}))['likes']
```

Comment:
I don't see why you have to do this rather than just `not unlike`. I think we should do what I suggested above so we can just directly pass the `like` parameter, which would be even simpler.